components/third_party/adblock/lists/easylist-cookie/README.chromium

```diff
@@ -0,0 +1,4 @@
+Name: EasyList Cookie
+URL: https://easylist.to/
```

Comment:
When there's a repository on GitHub or https://hg.adblockplus.org/, I tried to link to that instead thinking it might be more useful than either the text list or the front page of easylist.to.

In this case, is https://github.com/easylist/easylist/tree/master/easylist_cookie the place where this list is developed?

Reply:
Okay, done @fmarier